package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-red-contrib-focas",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
Since we are adding a new feature to the node, we should increment the minor version here instead of the patch